### ISSUE TITLE
branchLabelMapping 1:n mapping

### ISSUE THREAD

**Issue by MstrDC** [sqren/backport#445]
Hello,

Playing around to get the backport github action working.
I am wondering if the following setup is supporter ( I guess currently not)
`    "branchLabelMapping": {
        "master": "master",
        "2022.2": "release_2022.2",
        "2022.2": "release_2022.2AAA",
        "2022.2": "release_2022.2BBB"
    },`

What I am seeing is that 1 label will be executed for only a single line in this list.
Any idea how I would get this working?
If not, would this be a feature you want in your tool?

Additionally I am wondering what will happen when there is a merge conflict.
Will it create the PR which people can checkout and fix? Or will it simple log an error and not create a PR?

**Comment by sqren** [comment]
Hi there,

I think I can add support for this like:

```json
"branchLabelMapping": {
        "master": "master",
        "2022.2": ["release_2022.2", "release_2022.2AAA", "release_2022.2BBB"]
},
```

WDYT?

**Comment by MstrDC** [comment]
@sqren ,
Looks good to me.
If for each label, it is merged to all listed branches and it doesnt break the current regex implementation?

**Comment by sqren** [comment]
> If for each label, it is merged to all listed branches and it doesnt break the current regex implementation?

Yes, that was my intention. Haven't had time to look into it yet but hope to do it soon-ish
